.ci/Jenkinsfile

```diff
@@ -144,10 +144,10 @@ def generateTestCommandStage(Map args = [:]){
     withNode(labels: "ubuntu-18 && immutable", sleepMax: 20, forceWorkspace: true) {
       cleanup()
       try {
-        withMageEnv(){
+        dir("${BASE_DIR}"){
```

Comment:
It looks like changing the order of execution in this stub, changed path visibility?

Reply:
Aha, it uses the worker go version maybe, let me put a fix and run `dir` in conjunction with `withMageEnv`